staticaddr/protocol_version.go

```diff
@@ -0,0 +1,48 @@
+package staticaddr
+
+import (
+	looprpc "github.com/lightninglabs/loop/swapserverrpc"
+)
+
+// AddressProtocolVersion represents the protocol version (declared on rpc
+// level) that the client declared to us.
+type AddressProtocolVersion uint32
+
+const (
+	// ProtocolVersion_V0 indicates that the client is a legacy version
+	// that did not report its protocol version.
+	ProtocolVersion_V0 AddressProtocolVersion = 0
+
+	// stableRPCProtocolVersion defines the current stable RPC protocol
+	// version.
+	stableRPCProtocolVersion = looprpc.StaticAddressProtocolVersion_V0
```

Comment:
undefined references here and below, missing `go.mod` entry?

Reply:
Hmm ya, they will be used in a loop follow-up PR, but used meanwhile in a different project.